crates/bevy_ui/src/widget/text.rs

```diff
@@ -19,6 +22,27 @@ fn scale_value(value: f32, factor: f64) -> f32 {
     (value as f64 * factor) as f32
 }
 
+/// Text system flags
+///
+/// Used internally by [`measure_text_system`] and [`text_system`] to schedule text for processing.
+#[derive(Component, Debug, Clone, Reflect)]
+#[reflect(Component, Default)]
+pub struct TextFlags {
+    /// create a new measure for the text
+    remeasure: bool,
+    /// recompute the text
+    recompute: bool,
+}
```

Comment:
I feel like `remeasure` should be called `regenerate_measure_function` or similar. "measuring" is when you call the measure function not when you generate it. `recompute` could then be called `remeasure`, but I think `recompute` is also fine.

Reply:
Yeah the terminology everywhere needs to be tightened up. I've been loose with it because I'm not really 
clear on what's going to be intuitive for users etc, so just used short easy names. Even `regenerate_measure_function` is a bit misleading, since the flag is set when a new text node entity is spawned but its font hasn't loaded, so it is rescheduling the initial creation of a measure function to the next frame, not regenerating an existing measure function.

Reply:
Changed it to `generate_measure_function`, seems okay.

Reply:
Thought about this again, and these renamings are better as they capture the correct semantics I think:
* `generate_measure_func` -> `needs_new_measure_func`
* `recompute` -> `needs_recompute`